numpy/random/mtrand/mtrand.pyx

```diff
@@ -1732,17 +1749,17 @@ cdef class RandomState:
         """
         if high is None:
             warnings.warn(("This function is deprecated. Please call "
-                           "randint(1, {low} + 1) instead".format(low=low)),
+                           "randint(low=1, high={low} + 1) instead".format(low=low)),
```

Comment:
I also like the original better. Even after adding the keyword arguments, I think most numpy users are still going to use the positional version. Plus its more terse in this case.